pkg/docker/entrypoint.sh

```
@@ -225,55 +216,55 @@ function load_server_json_file() {
     # Validate PGADMIN_DEFAULT_EMAIL
     CHECK_EMAIL_DELIVERABILITY="False"
-    if [ -n "${PGADMIN_CONFIG_CHECK_EMAIL_DELIVERABILITY}" ]; then
-        CHECK_EMAIL_DELIVERABILITY=${PGADMIN_CONFIG_CHECK_EMAIL_DELIVERABILITY}
+    if [[ -n $PGADMIN_CONFIG_CHECK_EMAIL_DELIVERABILITY ]]; then
+        CHECK_EMAIL_DELIVERABILITY=$PGADMIN_CONFIG_CHECK_EMAIL_DELIVERABILITY
     fi
     ALLOW_SPECIAL_EMAIL_DOMAINS="[]"
-    if [ -n "${PGADMIN_CONFIG_ALLOW_SPECIAL_EMAIL_DOMAINS}" ]; then
-        ALLOW_SPECIAL_EMAIL_DOMAINS=${PGADMIN_CONFIG_ALLOW_SPECIAL_EMAIL_DOMAINS}
+    if [[ -n $PGADMIN_CONFIG_ALLOW_SPECIAL_EMAIL_DOMAIN ]]; then
+        ALLOW_SPECIAL_EMAIL_DOMAINS=$PGADMIN_CONFIG_ALLOW_SPECIAL_EMAIL_DOMAINS
```

Shellcheck (0.11.0)

[info] Possible misspelling: PGADMIN_CONFIG_ALLOW_SPECIAL_EMAIL_DOMAINS may not be assigned. Did you mean PGADMIN_CONFIG_ALLOW_SPECIAL_EMAIL_DOMAIN?

(SC2153)